### ISSUE TITLE
Set a comment agent

### ISSUE THREAD

**Issue by dougbeal** [aaronpk/OwnYourSwarm#26]
Bridgy sets a comment agent, which is handy for auto approving comments.  It would be great if OWS did this as well.
# Bridgy example:
```
Bridgy (https://brid.gy/about) AppEngine-Google; (+http://code.google.com/appengine; appid: s~brid-gy)
```

**Comment by aaronpk** [comment]
What is a comment agent? Like an HTTP User Agent header?

**Comment by dougbeal** [comment]
I don't know, I guess it must be a wordpressism.
Perhaps @snarfed can enlighten us, other wise I will dive in and see where it comes from.

**Comment by snarfed** [comment]
yes, that looks like HTTP User-Agent.

@dougbeal, you might just consider using this PHP snippet to auto-approve all webmentions, since as far as we know, we haven't even seen a single webmention spam in the wild yet.

alternatively, you could also auto-approve by source URL domain, ie ownyourswarm.p3k.io in this case.

**Comment by dshanske** [comment]
Does OYS set a specific user agent?

**Comment by aaronpk** [comment]
Closed in e72fadf0dc7fe6378b4521dd42d0b715697b9d56
